### PR TITLE
Lots of fixes to pl translation

### PR DESCRIPTION
**Pull request by Friskygote**
Apologize for making another PR, I'm absolutely bad at my job. Added a lot of fixes to typos, added new translations, improved existing ones.

### CODE REVIEW

**Comment by rail01** [comment]
Bad goat. Bad!

File: i18n/pl.json [hunk 1]
```diff
@@ -178,19 +178,20 @@
 			["content_team_member", "Członek Zespołu Treści"],
 			["GRASP", "GRASP"],
 			["bot-global", "bot Fandom"],
-			["staff", "personel Fandom"],
+			["staff", "personel Fandomu"],
 			["wiki-manager", "Menedżer wiki"],
-			["helper", "pomocnik Fandom"],
+			["helper", "pomocnik Fandomu"],
```
Comment (rail01) [inline i18n/pl.json:183]: Fandom Staff and Helpers groups aren't localized and neither they should be. Please leave it as "Fandom Staff" for Staff members and "Helper" for Helpers.

**Comment by Friskygote** [comment]
> Good now

thank you my lord

File: i18n/pl.json [hunk 2]
```diff
@@ -9,24 +10,24 @@
 		"strona": "page",
 		"szukaj": "search",
 		"🎲": "random",
-		"discussions": "discussion",
```
Comment (Markus-Rost) [inline i18n/pl.json:12]: That would remove the `discussions` alias for the `discussion` command. It should probably stay to keep English commands working in all languages. You can add a new key for other Polish alias instead.

Comment (Friskygote) [reply i18n/pl.json:12]: i feel bullied by two people at this point

File: i18n/pl.json [hunk 3]
```diff
@@ -10,23 +11,24 @@
 		"szukaj": "search",
 		"🎲": "random",
 		"discussions": "discussion",
+                "dyskusje": "discussion",
```
Comment (Markus-Rost) [inline i18n/pl.json:14]: please use tabs instead of spaces

Comment (Markus-Rost) [reply i18n/pl.json:14]:
```suggestion
		"dyskusje": "discussion",
```

**Pull request merged**